packages/amplify/amplify_flutter/example/integration_test/config_test.dart

```diff
@@ -71,7 +71,6 @@ const amplifyconfig = ''' {
                 "Auth": {
                     "Default": {
                         "authenticationFlowType": "USER_SRP_AUTH",
-                        "loginMechanisms": [],
```

Comment:
*Not in scope for your PR*

@dnys1 - I see this was added in https://github.com/aws-amplify/amplify-flutter/pull/1011. I assume these values are either fake, or are from an app that no longer exists. We should make that clear (a comment would be fine) because this could set off some false alarms.

Reply:
Yeah, good point. I'll clean this all up in a separate PR.